tests/monero_wallet_rpc.go

```diff
@@ -28,6 +28,7 @@ func CreateWalletRPCService(t *testing.T) string {
 	t.Cleanup(func() {
 		_ = outPipe.Close()
 		_ = cmd.Process.Kill()
+		_ = cmd.Wait()
```

Comment:
The defunct processes were getting cleaned up when the test of a particular package ended, so  this change isn't critical, but I should have been reaping the exited children immediately.